Formula/snappy.rb

```diff
@@ -17,11 +17,28 @@ class Snappy < Formula
   depends_on "cmake" => :build
   depends_on "pkg-config" => :build
 
+  # Fix issue where Mojave clang enters a __GNUC__ block
+  if MacOS.version <= :mojave
+    depends_on "gcc"
+    fails_with :clang do
+      cause "error: invalid output constraint '=@ccz' in asm"
+    end
+  end
+
+  # Fix issue where `snappy` setting -fno-rtti causes build issues on `folly`
+  # `folly` issue ref: https://github.com/facebook/folly/issues/1583
+  patch :DATA
+
   def install
-    system "cmake", ".", *std_cmake_args
+    args = *std_cmake_args + %w[
```

Comment:
```suggestion
    args = std_cmake_args + %w[
```
I don't think we need to destructure the array here.